ui/src/stores/HomeStore.js

```diff
@@ -403,12 +406,14 @@ class HomeStore {
 
   @action
   async filterByTxHashInReturnValues(transactionHash) {
-    const events = await this.getEvents(1, 'latest')
+    const { blockNumber } = await this.homeWeb3.eth.getTransaction(transactionHash)
```

Comment:
What if the transaction is not mined yet?

Reply:
Indeed, there was a mistake. I have fixed it, and checked the ui in local testnet, everything seems be working fine.